sechub-doc/src/docs/asciidoc/documents/shared/configuration/go-gen-supported-source-extensions.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+SOURCE=supported-source-extensions.txt
+
+# This script is intended to generate the go code snippet for constants.go
+# - step 1: edit $SOURCE
+# - step 2: run this script and copy the output
+# - step 3: replace the DefaultZipAllowedFilePatterns declaration in constants.go
+
+cd `dirname $0`
```

Comment:
I am not sure if it is a good idea to trust the user input unconditionally.

Reply:
there is no user input here;
The code just changes into the directory where the script is located. This is convenient when calling the script from a Gui or from another working directory.